src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx

```diff
@@ -0,0 +1,50 @@
+import { Certificate, ProjectExpense } from '@planet-sdk/common';
+import styles from '../styles/ProjectInfo.module.scss';
+import { useTranslations } from 'next-intl';
+import SingleProjectInfoItem from './microComponents/SingleProjectInfoItem';
+import ExternalCertificationItem from './microComponents/ExternalCertificationItem';
+import ProjectSpendingItem from './microComponents/ProjectSpendingItem';
+
+interface ExternalCertificationProps {
+  certification: Certificate[];
+  spendings: ProjectExpense[];
+}
```

Comment:
1. Let's match the prop names for ProjectDownloads with those returned in the API. Go down the component heirarchy while updating the names.
2. Prop interface should be just `Props` or `ProjectDownloadsProps` instead of the `ExternalCertificationProps` which is only a part of the functionality.